paddle/fluid/inference/api/paddle_analysis_config.h

```diff
@@ -23,8 +23,9 @@
 
 // Here we include some header files with relative paths, for that in deploy,
 // the abstract path of this header file will be changed.
-#include "paddle_api.h"           // NOLINT
-#include "paddle_pass_builder.h"  // NOLINT
+#include "paddle_api.h"               // NOLINT
+#include "paddle_pass_builder.h"      // NOLINT
+#include "paddle_quantizer_config.h"  // NOLINT
```

Comment:
```
#ifdef PADDLE_WITH_MKLDNN
#include "paddle_quantizer_config.h" 
#endif 
```

Reply:
I think you can add `#ifdef`s. Besides, would you create a new clean PR for C-API quantization core? This PR contains a lot of information.

Reply:
OK, I will create a new PR.